test/e2e/pods.go

```diff
@@ -289,29 +291,38 @@ var _ = Describe("Pods", func() {
 		pods, err := podClient.List(labels.SelectorFromSet(labels.Set(map[string]string{"time": value})), fields.Everything())
 		Expect(len(pods.Items)).To(Equal(1))
 
-		By("retrieving the pod")
-		podOut, err := podClient.Get(pod.Name)
-		if err != nil {
-			Fail(fmt.Sprintf("Failed to get pod: %v", err))
-		}
+		// Standard get, update retry loop
+		expectNoError(wait.Poll(time.Second*5, time.Minute*2, func() (bool, error) {
```

Comment:
No need to wait 5 whole seconds between each attempt, is there? Can we reduce this to every 500ms for up to 30s?